spec/controllers/admin/bulk_line_items_controller_spec.rb

```diff
@@ -373,21 +376,22 @@
 
       it "correctly updates order totals and states" do
         expect(order.total).to eq 35.0
+        expect(order.shipment_adjustments.shipping.sum(:amount)).to eq 6.0
+        expect(order.shipment_adjustments.tax.sum(:amount)).to eq 0.29
         expect(order.item_total).to eq 20.0
         expect(order.adjustment_total).to eq 15.0
         expect(order.included_tax_total).to eq 1.22
         expect(order.payment_state).to eq "paid"
 
-        expect(order).to receive(:update_order!).at_least(:once).and_call_original
-        expect(order).to receive(:create_tax_charge!).at_least(:once).and_call_original
-
         spree_delete :destroy, params
         order.reload
 
-        expect(order.total).to eq 22.0
+        expect(order.total).to eq 19.0
+        expect(order.shipment_adjustments.shipping.sum(:amount)).to eq 3.0
+        expect(order.shipment_adjustments.tax.sum(:amount)).to eq 0.14
         expect(order.item_total).to eq 10.0
-        expect(order.adjustment_total).to eq 12.0
-        expect(order.included_tax_total).to eq 0.99
+        expect(order.adjustment_total).to eq 9.0
+        expect(order.included_tax_total).to eq 0.84
```

Comment:
Are these the changes to the totals that correct the bug that's currently in production?

Reply:
Shockingly, it shows an entirely separate bug which wasn't even documented (which is fixed here). The shipping fee was not being updated correctly, but it is now.